Yoast/javascript
# FRO-16 Shared button styling

## Summary


This PR can be summarized in the following changelog entry:

* [@yoast/components] Changes the table navigation button component to be inline with the new Yoast styling.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch
* Checkout `14297-update-styling-table` from the wordpress-seo-premium repo
* Link the javascript repo
* Build the plugin
* Check the page navigation buttons of the tables if they are correct and inline with the new Yoast styling.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes #
